### PR TITLE
Password strength while changing password

### LINKED ISSUE #545: Add password strength in settings page
**Actual Behaviour**

While entering the password in the settings page(changing password) there is no password strength indicator.

**Expected Behaviour**

While entering the password in the settings page(changing password) the password strength is shown.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #545 

Changes: While entering the password in the settings page(changing password) the password strength is shown.
Surge Deployment Link: https://pr-557-fossasia-susi-accounts.surge.sh

Screenshots for the change:
pass-set-strgnt

### CODE REVIEW

**Comment by samagragupta** [comment]
@rossoskull Done the required changes.

**Comment by samagragupta** [comment]
@PrP-11 Done the required changes.

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 1]
```diff
@@ -134,6 +138,15 @@ export default class ChangePassword extends Component {
           this.confirmPasswordErrorMessage = '';
           this.newPasswordErrorMessage = '';
         }
+        if (!state.newPasswordError) {
+          let result = zxcvbn(newPassword);
+          state.newPasswordScore = result.score;
+          let strength = ['Worst', 'Bad', 'Weak', 'Good', 'Strong'];
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: const

**Comment by AakashMallik** [comment]
@samagragupta There is a lot of scope for basic refactoring in the code. It would be really nice if you could use ```const``` where ever possible. Use destructuring where ever needed and simplify function logic as much as possible.

File: src/components/Auth/ChangePassword/ChangePassword.react.js (hunk 1 above)
Comment (samagragupta) [reply src/components/Auth/ChangePassword/ChangePassword.react.js]: Done the changes

**Comment by samagragupta** [comment]
@mariobehling @Akshat-Jain Please review

**Comment by AakashMallik** [comment]
@samagragupta Squash commits

**Comment by samagragupta** [comment]
@anshumanv In my case it is proper.
password

**Comment by anshumanv** [comment]
Not so sure about this then, meanwhile please fix the unequal fields sizing issue while you're at it. 💯

**Comment by anshumanv** [comment]
@neeraj3029 @rossoskull @AakashMallik @fragm3 can you check consequently?

**Comment by AakashMallik** [comment]
@anshumanv It ain't spilling out in my case, but then again, as soon as I start typing, the input box shrinks. 

screenshot-pr-557-fossasia-susi-accounts surge sh-2018 12 05-20-46-03

**Comment by AakashMallik** [comment]
@samagragupta Could you please fix this shrinking input box?

**Comment by samagragupta** [comment]
@akshatnitd @mariobehling Please review

**Comment by samagragupta** [comment]
@akshatnitd @anshumanv Please review it.

**Comment by samagragupta** [comment]
@anshumanv @akshatnitd Please review.

**Pull request merged**